src/uu/factor/src/cli.rs

```diff
@@ -34,14 +33,21 @@ fn print_factors_str(num_str: &str, w: &mut impl io::Write) -> Result<(), Box<dy
 }
 
 pub fn uumain(args: impl uucore::Args) -> i32 {
-    let matches = app!(SYNTAX, SUMMARY, LONG_HELP).parse(
-        args.collect_str(InvalidEncodingHandling::Ignore)
-            .accept_any(),
-    );
+    let matches = App::new(executable!())
+        .version(VERSION)
+        .about(SUMMARY)
+        .arg(Arg::with_name("NUMBER").multiple(true))
```

Comment:
Just a small nitpick, for consistency can you use options like this?

https://github.com/uutils/coreutils/blob/222bd8119083792a6468693bb56194500b4102e5/src/uu/ls/src/ls.rs#L61-L68

Reply:
Sure :) done